style.css

```diff
@@ -1,36 +1,28 @@
-
 @import url(https://fonts.googleapis.com/icon?family=Material+Icons);
 @import url(./components/components.css);
 
 html {
     scroll-behavior: smooth;
-}
-*{
     color: var(--OFF-BLACK);
-
 }
 
 /* DARK MODE */
 
-html[data-theme="dark"] .box-shadow-100{
+html[data-theme="dark"] .box-shadow-100 {
     box-shadow: 0 5px 2.5rem 2px var(--OFF-WHITE);
 }
 
-
-html[data-theme="dark"] .doc-body{
+html[data-theme="dark"] .doc-body {
     --COLOR-PRIMARY-50-SHADE-1: #d7f8e318;
     background-color: var(--OFF-WHITE);
 }
 
-html[data-theme="dark"] .landing{
+html[data-theme="dark"] .landing {
     --COLOR-PRIMARY-50-SHADE-1: #d7f8e318;
     background-color: var(--OFF-WHITE);
```

Comment:
you can combine the classes using `,` and give the properties, also try to declare variables in one place it's easy to keep track

Reply:
Yup sure will do that